.github/workflows/android-build.yml

```
@@ -0,0 +1,75 @@
+    - name: Start Xvfb
+      run: |
+        sudo apt-get install -y xvfb
+        Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
+    
+    - name: Set up Qt
+      run: |
+        # Cache directory for downloaded Qt
+        mkdir -p $HOME/Qt
+        export QT_ROOT=$HOME/Qt
+        export WORKDIR=$PWD
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)